src/pages-chibi/implementations/Atsumaru/main.ts

```diff
@@ -83,6 +83,25 @@ export const Atsumaru: PageInterface = {
         .ifNotReturn()
         .run();
     },
+    getMalUrl($c) {
+      const getMal = $c
+        .provider()
+        .equals('MAL')
+        .ifNotReturn()
+        .querySelector('.aspect-square[title="MyAnimeList"]')
+        .getAttribute('href')
+        .run();
+
+      const getAnilist = $c
+        .provider()
+        .equals('ANILIST')
+        .ifNotReturn()
+        .querySelector('.aspect-square[title="AniList"]')
+        .getAttribute('href')
+        .run();
+
+      return $c.coalesce($c.fn(getMal).run(), $c.fn(getAnilist).run()).ifNotReturn().run();
```

Comment:
```suggestion
      return $c.coalesce(getMal, getAnilist).ifNotReturn().run();
```

Try this, it should work. Unless it doesn't ? Previous implementation makes you run the code twice, technically.

Reply:
hmm 
<img width="570" height="29" alt="onsite replace" src="https://github.com/user-attachments/assets/2ac1047a-a50f-4385-bc13-df50a26c1782" />
it only work for first coalesce function, the second getAnilist doesn't run when I set the provider as anilist if i remove fn

Reply:
> you can't return all of those at once, it doesn't make sense the implementation doesn't handle multiple string, only one.

i mean, i'm using 1 provider per testing. when using myanimelist, it do return myanimelist when it show Overwrite in onsite url in console.

But when i use anilist as provider when testing, it doesn't return anything.